tuned/plugins/base.py

```diff
@@ -519,18 +526,20 @@ def _norm_value(self, value):
 			return re.sub(r'^\s*(0+,?)+', "", v)
 		return v
 
-	def _verify_value(self, name, new_value, current_value, ignore_missing, device = None):
+	def _verify_value(self, name, new_value, current_value, ignore_missing, device=None, iname=""):
 		if new_value is None:
 			return None
+
+		global dlog
 		ret = False
 		if current_value is None and ignore_missing:
 			if device is None:
 				log.info(consts.STR_VERIFY_PROFILE_VALUE_MISSING % name)
 			else:
 				log.info(consts.STR_VERIFY_PROFILE_DEVICE_VALUE_MISSING % (device, name))
-			return True
+			ret = True
```

Comment:
This change can result in line 559 calling the self._log_verification_result(), which will log additional PROFILE_VALUE_OK, which previously didn't happen.

Reply:
Fixed it.